ffbs-mesh-vpn-parker/files/usr/sbin/nodeconfig.sh

```diff
@@ -3,9 +3,10 @@
 # THIS SCRIPT MUST HAVE GID 800 (gluon-mesh-vpn)
 # USE `gluon-wan nodeconfig.sh`
 # !!!!!!
-SIGN_PUB_KEY=/etc/parker/node-config-pub.key
+SIGN_PUB_KEY=/tmp/parker/nodeconfig-pub.key
 DEFAULT_SLEEP=20
 export LIBPACKETMARK_MARK=1
+mkdir -p /tmp/parker/
 
 tmpdir=/tmp/ff-Ohb0ba0u/
```

Comment:
We should agree on one of the dirs in `/tmp/`. Currently (for a reason unclear to me) Parker uses `/tmp/ff-Ohb0ba0u/` everywhere.

I am fine with changing this to `/tmp/parker/` - but we should make sure that  we do it everywhere.

Reply:
I would suggest to keep `/tmp/ff-Ohb0ba0u` for now, to keep the diff small for now. We can still clean this name up in a later round.